runner/internal/shim/docker.go

```diff
@@ -169,38 +178,38 @@ func (d *DockerRunner) Run(ctx context.Context, cfg TaskConfig) error {
 		}
 	}()
 
-	d.state = Running
-	d.containerID = containerID
-	d.containerName = cfg.ContainerName
-	log.Printf("Running container, name=%s, id=%s\n", d.containerName, containerID)
+	log.Printf("Running container, name=%s, id=%s\n", task.containerName, containerID)
```

Comment:
Have you considered switching to a level-aware logging like logrus used in the runner?

Reply:
Logging facility revamping is in the plans, will be eventually submitted as a separate PR.